styles.css

```diff
@@ -0,0 +1,183 @@
+a {
+  text-decoration: none;
+}
+
+.header {
+  width: 375px;
+  height: 92px;
+  background-color: #303030;
+  display: flex;
+  justify-content: space-between;
+}
+
+.name {
+  color: white;
+  height: 24px;
+  font-family: Roboto, serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: center;
+  position: relative;
+  top: 20px;
+  left: 10px;
+}
+
+.name p {
+  height: 24px;
+  width: 187px;
+}
+
+.burger {
+  height: 24px;
+  width: 20px;
+  margin-left: 40px;
+  margin-right: 5px;
+  margin-bottom: 15px;
+  border-radius: 0;
+  align-self: center;
+  position: relative;
+  top: 10px;
+  right: 10px;
+}
+
+.burger a {
+  background-color: white;
+  margin-bottom: 5px;
+  padding-bottom: 0;
+}
+
+.main {
+  border-top: 1px solid slateblue;
+  color: white;
+  background-color: #151515;
+  width: 375px;
+  height: 610px;
+}
+
+.title {
+  margin-left: 20px;
+}
+
+.head {
+  margin-left: 20px;
+}
+
+.content {
+  margin-left: 20px;
+}
+
+h1 {
+  border-left: 20px;
+  color: #ff6b00;
+  height: 124px;
+  width: 327px;
+  left: 24px;
+  top: 120px;
+  font-family: Crete Round, serif;
+  font-size: 56px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  text-align: left;
+}
+
+.pretext {
+  height: 37px;
+  width: 327px;
+  left: 24px;
+  top: 256px;
+  font-family: Roboto, serif;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.text {
+  height: 100px;
+  width: 327px;
+  left: 24px;
+  top: 256px;
+  font-family: Poppins, serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+nav {
+  height: 49px;
+  width: 327px;
+  left: 24px;
+  top: 256px;
+  display: flex;
+  justify-content: space-between;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.bg {
+  width: 30px;
+  height: 20px;
+}
+
+#menu {
+  display: flex;
+  flex-direction: row;
+  width: 170px;
+  height: 26px;
+}
+
+.github {
+  height: 20px;
```

Comment:
- [ ] I can see that this property ` height: 20px;` is been repeated severally in your CSS. Why not create a class for this repeated property `height: 20px;` and then add it to the `HTML` elements that use the property? This will help you to achieve clean code and not to repeat yourself over again. checkout this `DRY` [principle](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md)

Reply:
Done.